modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -91,6 +93,19 @@ def veteran_from_headers(with_gender: false)
         vet
       end
 
+      def validate_ccg_token!
+        client = TokenValidation::V2::Client.new(api_key: Settings.token_validation.api_key)
```

Comment:
The setting will have to be changed to :: `Settings.claims_api.token_validation.api_key` to match what has been stored in the DEVOPS repo.